dev/Prototypes/ItemsViewPrototype/Control/ViewDefinitions/Table/ItemsViewColumnDefinitionBase.cs

```diff
@@ -7,6 +7,8 @@ namespace DEPControlsTestApp.ItemsViewPrototype
     // Would implementations of this base type want to add dependency properties?
     public class ItemsViewColumnDefinitionBase: DependencyObject
     {
+        public virtual string ColumnName { get; set; }
```

Comment:
Is this just for identification purposes ?

Reply:
yes, precisely. If you follow along the other changes, you will see that in FileExplorerScenario.xaml.cs, we just assumed that all columns were ItemsViewTextColumnDefinitions. Then we were using the DisplayMemberPath to identify the columns. DisplayMemberPath is the binding path, as well as the heading text, unless overridden by heading. The change I made was so that we could identify a column at the base class level, without assuming it is a TextColumn. Then you will also see that ItemsViewTextColumnDefinition defaults to using DisplayMemberPath as the ColumnName, if nothing was set for it directly, similar to how DisplayMemberPath is used for Heading.